File: Insertion Sort/insertion_sort.js
const InsertionSort = (arr) => {
  for (let i = 1; i < arr.length; ++i)
    for (let j = i; j >= 1; --j) {
      if (arr[j - 1] <= arr[j]) break;
      [arr[j], arr[j - 1]] = [arr[j - 1], arr[j]];
    }
  return arr;
};

File: Insertion Sort/insertion_sort.cs
internal static int[] insertion_sort(int[] arr)
{
    for(int i = 1; i < arr.Length; ++i)
    {
        for(int j = i; j >= 1; --j)
        {
            if (arr[j - 1] <= arr[j])
                break;
            int temp = arr[j];
            arr[j] = arr[j - 1];
            arr[j - 1] = temp;
        }
    }
    return arr;
}